quarkus/extensions/kogito-quarkus-processes-extension/kogito-quarkus-processes-deployment/src/main/java/org/kie/kogito/quarkus/processes/deployment/ProcessesAssetsProcessor.java

```diff
@@ -123,7 +123,7 @@ public void postGenerationProcessing(
         // configure the application generator
         KogitoBuildContext context = kogitoBuildContext(root.getPaths(), aggregatedIndex, curateOutcomeBuildItem.getEffectiveModel().getAppArtifact());
 
-        Collection<GeneratedFile> generatedFiles = generatePersistenceInfo(
+        Collection<GeneratedFile> generatedPersistenceFiles = generatePersistenceInfo(
```

Comment:
What about wrapping with `new ArrayList`?
```suggestion
        Collection<GeneratedFile> generatedFiles = new ArrayList<>(generatePersistenceInfo(
```
In this case you can remove the other change and keep the `.addAll(` code

Reply:
Fine with me.